### PR TITLE
Implement L2 Costs Aggregator Indexer

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-5073, L2B-5074

### CODE REVIEW

**Review by maciekop-l2b — APPROVED** [review]

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.test.ts [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+      indexer.shift = mockFn().returns([MIN, MIN])
+
+      // from 2023-05-01 00:00:00 to 2023-05-01 00:30:00
+      const result = await indexer.update(MIN.toNumber(), to.toNumber())
+
+      // 2023-05-01 00:30:00
+      expect(result).toEqual(to.toNumber())
+    })
+
+    it('throws if prices not available', async () => {
```
Comment (torztomasz) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.test.ts:92]: that should be the test of aggregate fn

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    await this.$.aggregatedL2CostsRepository.addMany(aggregated)
+    this.logger.info('Aggregated L2 costs', {
+      count: aggregated.length,
+    })
+
+    return shiftedTo.add(1, 'seconds').toNumber()
+  }
+
+  override async invalidate(targetHeight: number): Promise<number> {
+    //TODO: Check if targetHeight is inclusive
```
Comment (torztomasz) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.ts:62]: To resolve

Comment (maciekop-l2b) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.ts:62]: Resolved: it should be exclusive (`targetHeight` refers to the last entry that should remain)

Comment (torztomasz) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.ts:62]: Then we should change `deleteFrom (>=)` to `deleteAfter (>)`

Comment (maciekop-l2b) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsAggregatorIndexer.ts]: Done

**Pull request merged**